static/src/stylesheets/module/content/_article.scss

```diff
@@ -82,4 +82,9 @@
     .fc-container--has-show-more .treats__container {
         bottom: $gs-baseline;
     }
+
+    .fc-container--lazy-load {
+        display: block !important;
```

Comment:
better to avoid important if possible, but if it's just for a test it won't be worth worrying too much